app/controllers/rooms_controller.rb

```diff
@@ -180,6 +180,11 @@ def start
     opts[:mute_on_start] = room_setting_with_config("muteOnStart")
     opts[:require_moderator_approval] = room_setting_with_config("requireModeratorApproval")
     opts[:record] = record_meeting
+    opts[:voice_bridge] = @room_settings["voiceBridgePin"]
```

Comment:
This needs to be removed

Reply:
I have make changes for this please review it